client/src/pages/Home/MemberSays/MemberSays.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Cards from "./Cards";
+import "./Member.css";
+
+function MemberSays() {
+	return (
+		<div
+			style={{
+				backgroundColor: "#1E6DD0",
+				display: "flex",
+				flexDirection: "column",
+				alignContent: "center",
+				height: "75vh",
+				alignItems: "stretch",
+				gap: "40px",
+				padding: "5px",
```

Comment:
it's much better to give the div a class name then you can remove this inline style to the style file

Reply:
Done